### PR TITLE
[MIG] website_report : migrate to new api

### PR DESCRIPTION
**Pull request by atp-odoo**
TASK: https://www.odoo.com/web?db=openerp#id=12129&view_type=form&model=project.task&menu_id=3942&action=327
pad: https://pad.odoo.com/p/r.hjFLZuQTPIMeyiRP

### CODE REVIEW

File: addons/website_report/views/layouts.xml [hunk 1]
```diff
@@ -46,4 +46,4 @@
     </xpath>
 </template>
 </data>
-</openerp>
```
Comment (atp-odoo) [inline addons/website_report/views/layouts.xml]: Remove <data> </data> tag also. <odoo> is enough.

Comment (bba-odoo) [inline addons/website_report/views/layouts.xml]: done

**Comment by atp-odoo** [comment]
1) Reorder import from controller file and remove license from controller
2) one file report.py is missing to migrate.
3)rename views/layouts.xml - views/layouts_templates.xml with separate commit.

File: addons/website_report/models/report.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from openerp.http import request
+
+
+class Report(models.Model):
+    _inherit = 'report'
+
+    @api.v7
+    def translate_doc(self, cr, uid, doc_id, model, lang_field, template, values, context=None):
+        if request and hasattr(request, 'website'):
+            if not request.website:
```
Comment (atp-odoo) [inline addons/website_report/models/report.py]: wrong condition. if not request.website: then request.website.get_template ? 
it should be 
if request.website:
    v = request.website.get_template(template)

File: addons/website_report/models/report.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        return super(Report, self).translate_doc(cr, uid, doc_id, model, lang_field, template, values, context=context)
+
+    @api.v8
+    def translate_doc(self, model, lang_field, template, values):
+        return self._model.translate_doc(self._cr, self._uid, self.ids, model, lang_field, template, values, context=self._context)
+
+    @api.multi
+    def render(self, template, values=None):
+        if request and hasattr(request, 'website'):
+            if not request.website:
```
Comment (atp-odoo) [inline addons/website_report/models/report.py]: same as above
condition should be 
if request.website:

**Pull request closed without merging**